forui/lib/widgets.dart

```diff
@@ -0,0 +1,13 @@
+/// The Forui widgets and their corresponding styles.
+library forui.widgets;
+
+export 'src/widgets/badge/badge.dart' hide FBadgeContent, Variant;
+export 'src/widgets/button/button.dart' hide FButtonContent, Variant;
+export 'src/widgets/card/card.dart' hide FCardContent;
+export 'src/widgets/dialog/dialog.dart' hide FDialogContent, FHorizontalDialogContent, FVerticalDialogContent;
```

Comment:
I felt that it was better since it allowed us to test widgets that we don't necessary want to make public. Furthermore there's cases like `FTappable` which we want to make usable only within our own package.

Reply:
I don't think we test the content directly in this case. Generally, you would only want to test the public interface as well.